index.bs

```diff
@@ -505,9 +521,35 @@ The `:picture-in-picture` <a>pseudo-class</a> MUST match any <a>element</a>
 
 <em>This section is non-normative.</em>
 
-The API applies only to {{HTMLVideoElement}} in order to start on a minimal
-viable product that has limited security issues. Later versions of this
-specification may allow PIP-ing arbitrary HTML content.
+The {{HTMLVideoElement}} Picture-in-Picture API has limited security issues.
+
+## Restrictions on arbitrary content ## {#arbitrary-content-restrictions}
+
+The <a>Picture-in-Picture browsing context</a> MUST have top-level navigation
+blocked and the following <a>sandboxing flag set</a> applied:
+- <a>sandbox propagates to auxiliary browsing contexts flag</a>
+- <a>sandboxed auxiliary navigation browsing context flag</a>
+- <a>sandboxed document.domain browsing context flag</a>
+- <a>sandboxed forms browsing context flag</a>
+- <a>sandboxed modals flag</a>
+- <a>sandboxed navigation browsing context flag</a>
+- <a>sandboxed orientation lock browsing context flag</a>
+- <a>sandboxed pointer lock browsing context flag</a>
+- <a>sandboxed presentation browsing context flag</a>
+
+If the user agent restricts autoplay then this SHOULD be allowed by default.
+
+If the <a>Picture-in-Picture browsing context</a> is showing
+<a>non-interactive arbitrary content</a> then the user agent MUST not allow
+<a>user interaction</a>.
+
+If the <a>Picture-in-Picture browsing context</a> is showing <a>interactive
```

Comment:
QQ: How do web dev detect this restriction?

Reply:
I don't think they can. Maybe we could add a console message if a user tries to interact with a prohibited key?

Reply:
Maybe we can use KeyboardLayoutMap and remove any keys that are blocked (although this would have to be checked using JS running inside the PIP window):

https://developer.mozilla.org/en-US/docs/Web/API/KeyboardLayoutMap

Reply:
I'm not sure either how. The DevTools console message may help during development for sure.

Reply:
I have filed https://github.com/WICG/picture-in-picture/issues/143 to give this further thought